add-ons/example/example.addon

```diff
@@ -0,0 +1,9 @@
+# Name: example
+# Description: Shows basic functionality of an add-on
+
+# Aplication of the add-on
+# several commands can be used: echo, sleep, ssh, docker, oc, openshift
```

Comment:
We have Minishift documentation which covers [which commands are available](http://artifacts.ci.centos.org/minishift/minishift/docs/master/openshift-origin/latest/minishift/using/addons.html#addon-commands) in an add-on. Should we link to that instead of having this list to maintain in case of commands being added or removed in the future?